src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -61,6 +61,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
+        Private ReadOnly Property IIsInvalid As Boolean Implements IOperation.IsInvalid
```

Comment:
OK, that would be better. I'll make that change systematically in a near-future PR.